docs/products/opensearch/howto/control_access_to_content.md

```diff
@@ -1,93 +1,98 @@
+
+1. Open your Aiven for OpenSearch service in the [Aiven Console](https://console.aiven.io).
+1. Click <ConsoleLabel name="serviceusers"/> in the sidebar.
+1. Toggle the **Access Control** switch to **Enabled**
 
-## Create user with access control
+## Create a user with access control
 
-To create a service new user with access control:
+To create a service user with access control in Aiven for OpenSearch:
 
-1.  In your Aiven for OpenSearch service, select **Users** from the left
-    sidebar.
-2.  Select **Create user**.
-3.  In the **Create service user** screen, enter a **username**.
-4.  Specify an **index** pattern and set the desired **permissions**.
-5.  Add multiple rules to the user by selecting **Add another rule**.
-6.  Select **Save** to finish, and view the new user in the list.
+1. Open your Aiven for OpenSearch service in the [Aiven Console](https://console.aiven.io).
+1. Click <ConsoleLabel name="serviceusers"/> in the sidebar.
+1. Click **Create user**.
+1. In the **Create service user** screen, enter a **username**.
+1. Specify an **Index pattern** and set the desired **permissions**.
+1. To add multiple rules to the user, click **Add another rule**.
+1. Click **Save**.
 
 :::note
 The password for service users is automatically generated and can be
 reset if necessary.
 :::
 
-After creating a new service user, the next step is for them to log in
-to the
-[OpenSearch Dashboard](/docs/products/opensearch/dashboards) using their assigned credentials. This will grant them
-access to the dashboard, where they can perform various actions based on
-their assigned permissions.
+After creating a new service user, log in to the
+[OpenSearch Dashboard](/docs/products/opensearch/dashboards) using the assigned
+credentials to access and perform actions according to the user’s permissions.
 
 ## Manage users
 
-Aiven for OpenSearch provides several additional operations you can
-perform on service users, such as viewing, copying, resetting passwords,
-editing ACL rules, and deleting users.
+Aiven for OpenSearch provides several additional operations you can perform on
+service users.
 
-To access the additional operations, go to the **Users** tab
-within your Aiven for OpenSearch service, select the ellipsis (More
-options) in the respective user row, and choose the desired operation.
+To access these operations:
 
-:::note
-Deleting a service user will terminate all existing database sessions,
-and the user will lose access immediately.
+1. Open your Aiven for OpenSearch service in the [Aiven Console](https://console.aiven.io).
+1. Click <ConsoleLabel name="serviceusers"/> in the sidebar.
+1. Click <ConsoleLabel name="actions"/> in the respective user row and
+   choose the desired operation:
+
+   - Click <ConsoleLabel name="show password"/> to view the password.
+   - Click <ConsoleLabel name="reset password"/> to reset the password.
+   - Click <ConsoleLabel name="reset password"/> to reset the password.
```

Comment:
Is this a duplicate?

Reply:
It looks like reset is listed twice. :)

Reply:
Ahh, thanks! I didn't see that. I will update that in a new PR.